rest/utilities_testing.go

```diff
@@ -2337,11 +2337,23 @@ func GetNextContinuousChange(reader *bufio.Reader) (*db.ChangeEntry, error) {
 	}
 }
 
-func (rt *RestTester) GetChangesOneShot(t testing.TB, keyspace string, since int, username string, changesCount int) *TestResponse {
-	var changes struct {
-		Results []db.ChangeEntry
-		LastSeq db.SequenceID
+var changes struct {
+	Results []db.ChangeEntry
+	LastSeq db.SequenceID
+}
+
+// AssertChangesFeedMultiCollection Calls a changes feed on every collection and asserts that the nth expected change is
+// the number of changes for the nth collection.
+func (rt *RestTester) AssertChangesFeedMultiCollection(t testing.TB, username string, feedMode string, numKeyspaces int, expectedChanges []int, timeout int) {
```

Comment:
This LGTM but it's confusing to read so I'd just changes to

```
RequireContinousFeedChangesCount(tb testing.TB, username, keyspace, expectedChanges int) 
```

This function only works with continuous feed anyway, and then you can map keyspace name with number of changes.